src/components/form/input/FormInput.vue

```diff
@@ -6,11 +6,11 @@
         'form-input-warning': warning,
```

Comment:
Having the changes below does not show form input errors anymore:
![image](https://github.com/maevsi/maevsi/assets/4778485/6be3b4fe-3323-4a6c-a627-ef82d123f1c7)
See that `❌ Required` is missing. On master:
![image](https://github.com/maevsi/maevsi/assets/4778485/5993c942-23d2-47db-99ff-93bfafcecf0f)

Reply:
If you remove the required property the warning will also disappear. Maybe we can split it up? Or we can just ignore it for now and keep the required property.

Reply:
Huh, cannot reproduce anymore? Sus. But will resolve for now.